app/.env-example.env

```diff
@@ -13,3 +13,4 @@ NEXT_PUBLIC_CERAMIC_CLIENT_URL=https://ceramic-clay.3boxlabs.com
 
 NEXT_PUBLIC_DATADOG_APPLICATION_ID=some-uid-id
 NEXT_PUBLIC_DATADOG_CLIENT_TOKEN=somepubclienttoken
+NEXT_PUBLIC_DATADOG_ENV=someenvname
```

Comment:
unrelated but could you also change the example env `NEXT_PUBLIC_DPOPP_INFURA_KEY=YOUR_INFURA_KEY` -> `NEXT_PUBLIC_DPOPP_MAINNET_RPC_URL=YOUR_RPC_URL`